libs/components/src/button/button.component.ts

```diff
@@ -77,7 +78,16 @@ export class ButtonComponent implements ButtonLikeAbstraction {
 
   @Input() buttonType: ButtonTypes = null;
 
-  @Input() block?: boolean;
+  private _block?: boolean;
```

Comment:
We should also set a default value. Which should let us simplify the `block` check to just `this.block ? ["tw-inline-block] : ...`

```suggestion
  private _block = false;
```

Reply:
Apologies for missing that before. Updated!